app/hotels/src/HotelsContext.js

```diff
@@ -143,6 +150,47 @@ class Provider extends React.Component<Props, State> {
 
   setPaymentLink = (paymentLink: ?string) => this.setState({ paymentLink });
 
+  setCheckinDate = (checkin: Date) => {
```

Comment:
All this logic is because Booking.com only allows bookings for up to one month, correct?

Reply:
Correct